apps/authentication/api/face.py

```diff
@@ -185,7 +185,7 @@ def terminal_sessions(self):
         self._send_task("kill_session")
 
     def _send_task(self, task_name):
-        create_sessions_tasks(self.session_ids, 'Administrator', task_name=task_name)
+        create_sessions_tasks(self.session_ids, 'facelive', task_name=task_name)
 
 
 class FaceMonitorContextApi(CreateAPIView):
```

Comment:
I would need more context about the code to provide insights on any irregularities, potential issues or optimization suggestions.